components/css/mixins/_colors.scss

```diff
@@ -15,6 +15,10 @@ $color-mackeyOrange:#ff9b1a;
 $color-yellowWalk:#ffd100;
 $color-fountainRunTeal:#29a592;
 $color-pappysPurple:#ad1f65;
```

Comment:
Garrett defined these grays a while ago. I think they should make their way into production.